.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
```

Comment:
Is there anything to update by dependabot in this repository?

Reply:
Just for the github actions